bvm/ballerina-runtime/src/main/resources/MessagesBundle.properties

```diff
@@ -242,3 +242,6 @@ decimal.value.out.of.range = decimal range overflow
 config.size.mismatch = [{0}] the size for configurable variable ''{1}'' is expected to be ''{2}'', \
   but found ''{3}''
 invalid.utf8.byte.array.value = array contains invalid UTF-8 byte value
+negative.regexp.match.string.index = start index cannot be less than 0
```

Comment:
```suggestion
regexp.match.string.negative.index = start index cannot be less than 0
```

Reply:
Updated in https://github.com/ballerina-platform/ballerina-lang/pull/39444/commits/a7eb91d3a0dd754d2121774779e1e2d7fbe2b64e.